protos/feast/core/IngestionJob.proto

```diff
@@ -63,3 +63,14 @@ enum IngestionJobStatus {
     // job has been suspended
     SUSPENDED = 8;
 }
+
+message SpecsStreamingUpdateConfig {
```

Comment:
These proto messages definitely need comments on each field. It's quite useful since we will produce documentation from it here automatically: https://api.docs.feast.dev/grpc/feast.core.pb.html#IngestionJobStatus